src/output.rs

```diff
@@ -74,8 +74,18 @@ impl OutputType {
 
                 let is_less = pager_path.file_stem() == Some(&OsString::from("less"));
 
+                #[cfg(windows)]
+                let (pager_path, args) = {
+                    let p = std::env::var("ComSpec").unwrap_or_else(|_| "cmd".to_string());
+                    let mut a = args.to_vec();
+                    a.insert(0, pager_path.to_str().unwrap().to_string());
+                    a.insert(0, "/d/c".to_string());
+                    (p, a)
+                };
+
+                let mut p = Command::new(&pager_path);
+
```

Comment:
As explained in the top level comment, I'd prefer if we could revert this change.

Reply:
Will do...

Reply:
Done